Makefile

```diff
@@ -64,7 +64,7 @@ bindings-gen-go: build-contracts
 		type=$$(basename $$abi_file .abi.json); \
 		pkg=$$(basename $$type .sol | tr "[:upper:]" "[:lower:]"); \
 		mkdir -p ./bindings/go/$$pkg; \
-		abigen --abi $$abi_file --pkg $$pkg --type $$type --out ./bindings/go/$$pkg/$$type.go; \
+		abigen --abi $$abi_file --pkg $$pkg --type $$type --out ./bindings/go/$$pkg/$$type.go || exit 1; \
```

Comment:
yea, I can add the check for the command itself, but i think we should still keep this in case something else about the abigen command returns a non-zero exit